pkg/aws/elbv2/elbv2.go

```diff
@@ -207,20 +207,13 @@ func (e *ELBV2) DescribeTagsForArn(arn *string) (util.Tags, error) {
 // DescribeTargetGroupTargetsForArn looks up target group targets by an ARN.
 func (e *ELBV2) DescribeTargetGroupTargetsForArn(arn *string, targets ...[]*elbv2.TargetDescription) (result util.AWSStringSlice, err error) {
 	var targetHealth *elbv2.DescribeTargetHealthOutput
-	if len(targets) > 0 {
-		var mergedTargets []*elbv2.TargetDescription
-		for _, t := range targets {
-			mergedTargets = append(mergedTargets, t...)
-		}
-		targetHealth, err = e.DescribeTargetHealth(&elbv2.DescribeTargetHealthInput{
-			TargetGroupArn: arn,
-			Targets:        mergedTargets,
-		})
-	} else {
-		targetHealth, err = e.DescribeTargetHealth(&elbv2.DescribeTargetHealthInput{
-			TargetGroupArn: arn,
-		})
+	opts := &elbv2.DescribeTargetHealthInput{
+		TargetGroupArn: arn,
+	}
+	for _, targets := range targets {
+		mergedTargets = append(opts.Targets, target...)
```

Comment:
I think you mean `opts.Targets = append....`

Reply:
Whoops! ☕️

Reply:
Bah, I did a `go build cmd` but I guess that wouldn't have caught it anyway...